### ISSUE TITLE
Sailor + OpenResty

### ISSUE THREAD

**Issue by VoittajaTomi** [sailorproject/sailor#117]
I was wondering, why haven't there been a guide provided on how to install Sailor for Openresty? Seems to make more sense to use Openresty than Lighthttpd.

**Comment by Etiene** [comment]
You are right! We certainly need to improve our docs! For now, in the OS X setup you can find some information concerning Sailor and openresty: https://github.com/sailorproject/sailor/blob/master/docs/INSTALL_MAC.md#alternative-setup-using-nginx-and-openresty

**Comment by felipedaragon** [comment]
Which OS do you use, @VoittajaTomi?

**Comment by VoittajaTomi** [comment]
Hello, I use Fedora linux.

**Comment by ntfs32** [comment]
I think make a server nginx or openresty has more sense,but some thing wrong, with your gudie nginx_lua can not found sailor : 
no field package.preload['sailor']...
